charts/nri-prometheus/templates/configmap.yaml

```diff
@@ -9,10 +9,17 @@ data:
   config.yaml: |
     cluster_name: {{ include "nri-prometheus.cluster" . }}
 {{- if .Values.config }}
-{{- if (include "nri-prometheus.lowDataMode" .) }}
-{{ $lowDataDefault := .Files.Get "static/lowdatamodedefaults.yaml" | fromYaml -}}
-    {{- mergeOverwrite .Values.config $lowDataDefault | toYaml | indent 4 -}}
-{{- else }}
-    {{ toYaml .Values.config | indent 4 | trim }}
+    {{ omit .Values.config "transformations" | toYaml | indent 4 | trim -}}
+{{ if .Values.config.transformations }}
+    transformations: 
+    {{ .Values.config.transformations | toYaml | indent 4 | trim -}}
+{{- end }}
+{{- if (include "nri-prometheus.lowDataMode" .) -}}
+    {{ $lowDataDefault := .Files.Get "static/lowdatamodedefaults.yaml" | fromYaml }}
+    {{ if .Values.config.transformations }}
+    {{- $lowDataDefault.transformations | toYaml | indent 4 | trim }}
+    {{ else }}
+    {{- $lowDataDefault | toYaml | indent 4 | trim }}
+    {{ end }}
```

Comment:
This could lead to two `transformations` in case of `transformations` in the config and `lowDataMode` enabled.

A `merge` with the defaults (in case of `lowDataMode`) with the one that came from `.Values.configtransformations` should be needed or a way to allow overriding `transformations` properly.